week-5/elephant-talk/fill_data_csv.rb

```diff
@@ -0,0 +1,67 @@
+require "csv"
+
+class FillData
+  $file_products_customers = "products_customers.sql"
+  $file_transactions = "transactions.sql"
+  def products_customers
+    products_name = []
+    products_days = []
+    products_price = []
```

Comment:
Your way to store the different values is good, but you could have used only one variable array where each element would be another array and each element of that would be a field of your table.

Reply:
I think our way to store different values is simpler and more readable